addons/debugger/module.js

```diff
@@ -407,4 +409,37 @@ export const setup = (_vm) => {
     }
     return count;
   };
+
+  // All instances of AudioEngine use the same AudioContext by default,
+  // which means that opening the sound library resumes the VM's context. See #6847.
+  // This can be fixed by creating a separate context from the sound library.
+  addon.tab.waitForElement("[class*='play-button_play-button_']").then(() => {
```

Comment:
This needs to be optimized to avoid running `document.querySelector()` on every DOM changes.
We should check if any redux events are dispatched when opening the sound library. If not, we can pollute a relevant method that is called when the sound library is opened, and directly use querySelector there.
(I assume this waits until the sound library is opened, right? I haven't checked)

Reply:
`scratch-gui/modals/OPEN_MODAL` is probably the right Redux event.

Reply:
I added that reduxEvent.